src/Module.Server/Common/Models/CrpgItemValueModel.cs

```diff
@@ -83,14 +83,15 @@ private float CalculateArmorTier(ArmorComponent armorComponent)
             + 1.1f * armorComponent.BodyArmor
             + 1.15f * armorComponent.ArmArmor
             + 0.75f * armorComponent.LegArmor;
-        float bestArmorPower = 0.931878f * armorComponent.Item.ItemType switch
+
+        float bestArmorPower = armorComponent.Item.ItemType switch
         {
-            ItemObject.ItemTypeEnum.HeadArmor => 21.8288f,
-            ItemObject.ItemTypeEnum.Cape => 11.36f,
-            ItemObject.ItemTypeEnum.BodyArmor => 19.492481f,
-            ItemObject.ItemTypeEnum.HandArmor => 10.1992f,
-            ItemObject.ItemTypeEnum.LegArmor => 6.94f,
-            ItemObject.ItemTypeEnum.HorseHarness => 60f,
+            ItemObject.ItemTypeEnum.HeadArmor => 21.8288f * 0.931878f,
+            ItemObject.ItemTypeEnum.Cape => 11.36f * 0.931878f,
+            ItemObject.ItemTypeEnum.BodyArmor => 19.492481f * 0.931878f,
+            ItemObject.ItemTypeEnum.HandArmor => 10.1992f * 0.931878f,
+            ItemObject.ItemTypeEnum.LegArmor => 6.94f * 0.931878f,
+            ItemObject.ItemTypeEnum.HorseHarness => 45f * 1.1f,

```

Comment:
Hmm kind of weird to multiply 2 magic values, why not merge them?

Reply:
Yes I know. The reason is PAAAAAIN

Reply:
done